types/lowdb/lowdb-tests.ts

```diff
@@ -124,37 +118,32 @@ const weDidNotBreakLodash: ExampleSchema["posts"] = lodashChain
   const adapterLS = new LocalStorage<DbSchema>("test.json");
   const dbFP = lowfp(adapterLS);
   // Get posts
-  const postsFP: low.FpReturn<Post[], low.AdapterSync<DbSchema>> = dbFP(
-    "posts"
-  );
+  const postsFP = dbFP("posts");
 
   // replace posts with a new array resulting from concat
   // and persist database
   const write: Post[] = postsFP.write(
-    concat({ title: "lowdb is awesome", views: random(0, 5) })
+    concat<Post>({ title: "lowdb is awesome", views: random(0, 5) })
   );
 
   // Find post by id
-  const post: Post = postsFP(find({ id: 1 }));
+  const post: Post | undefined = postsFP(find<Post>({ id: 1 }));
 
   // Find top 5 fives posts
   const popular: Post[] = postsFP([
-    sortBy("views") as PostsAction,
+    sortBy<Post>("views") as PostsAction,
     take(5) as PostsAction
   ]);
 
-  const filtered: Post[] = dbFP("posts")(filter({ published: true }));
-  const writeAction: Post[] = dbFP("posts").write(concat({ id: "123" }));
-  const writeAction2: string = dbFP(["user", "name"]).write(set("typicode"));
+  const filtered: Post[] = dbFP("posts")(filter<Post>({ published: true }));
+  const writeAction: Post[] = dbFP("posts").write(concat<Post>({ id: 123 }));
+  const writeAction2: string = dbFP(["user", "name"]).write(() => "typicode");
```

Comment:
The documentation says to use `.write(set("typicode"))` here, but I think the documentation is wrong. Typescript generates an error if you do this, and rightly so in my opinion - doing this results in no value being written. `set()` is called internally by the `write()` function, so it shuldn't be needed.